src/routes/fetchMetricsData.ts

```diff
@@ -0,0 +1,31 @@
+import { validate, Joi } from 'express-validation';
+import UserReturnData from '../UserReturnData';
+import StatusCodes from '../StatusCodes';
+
+module.exports = (app: any, dbHandler: any) => {
+  const validation = {
+    query: Joi.object({
+      competition: Joi.string().required(),
+      team_number: Joi.string().required(),
```

Comment:
Property should be named `team` not `team_number` (Jacob's spec is incorrect). 

Should be of type number, not string

```suggestion
      team: Joi.number().required(),
```